pages/linux/systemctl-list-dependencies.md

```diff
@@ -0,0 +1,29 @@
+# systemctl list-dependencies
+
+> Show a unit's dependency tree in systemd.
+> See also: `systemctl list-units` to list loaded units.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemctl.html#list-dependencies%20UNIT%E2%80%A6>.
```

Comment:
```suggestion
> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#%0A%20%20%20%20%20%20%20%20%20%20%20%20list-dependencies%0A%20%20%20%20%20%20%20%20%20%20%20%20UNIT...%0A%20%20%20%20%20%20%20%20%20%20>.
```

Reply:
I would suggest `https://www.freedesktop.org/software/systemd/man/latest/systemctl.html`. The very weird and long postfix didn't add anything to me when visiting. Both just opens the same page and at the top of the page.